cves/CVE-2002-0840.yml

```diff
@@ -3,22 +3,22 @@ CWE_instructions: |
   Please go to cwe.mitre.org and find the most specific, appropriate CWE entry
   that describes your vulnerability. (Tip: this may not be a good one to start
   with - spend time understanding this vulnerability before making your choice!)
-CWE:
+CWE: CWE-20 Improper Input Validation
```

Comment:
Please just keep the cwe-20 and not the name

Reply:
Thanks for the feedback I'll make the changes right away